Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -30,7 +30,7 @@ final class SettingsAccountViewController: UIViewController, MessageBannerViewCo
     self.tableView.register(nib: .SettingsTableViewCell)
     self.tableView.register(nib: .SettingsAccountWarningCell)
     self.tableView.registerHeaderFooter(nib: .SettingsHeaderView)
-
+    self.tableView.registerHeaderFooterClass(SettingsGroupedFooterView.self)
```

Comment:
Can we keep the space between registering reusable views and calling `viewDidLoad` on the view model?